examples/tests/test_02_message_consumer.py

```diff
@@ -0,0 +1,127 @@
+"""
+Test Message Pact consumer.
+
+Pact was originally designed for HTTP interactions involving a request and a
```

Comment:
provide link to worth linking to https://docs.pact.io/getting_started/how_pact_works#non-http-testing-message-pact

and mentioning MessagePact is only responsible for the message, not the transport?